packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -81,6 +82,12 @@ export default async function processDeployment({
 
       if (event.type === 'created') {
         now._host = event.payload.url;
+
+        if (!quiet) {
+          log(`${event.payload.url} ${chalk.gray(`[v2]`)} ${deployStamp()}`);
```

Comment:
```suggestion
          log(`https://${event.payload.url} ${chalk.gray(`[v2]`)} ${deployStamp()}`);
```

Reply:
@rdev I'm pretty sure this needs to have https:// prefix to be backwards compatible with what it shows on stable. I'm not sure if the v1 also needs it.

Reply:
@styfle It's part of the payload now. This was causing the double `https://`

Reply:
I tried your branch again and I see the following:

```
> Deploying ~/Code/foo/node-try-catch under now-builders-ci-bot-85
> Using project node-try-catch
> node-try-catch-r4gd5xso2.now.sh [v2] [2s]
```

Notice how there is no https prefix. This is different behavior than `now@16.1.3`.

When I changed this line, it showed the correct output.